bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/persistence/PersistenceResource.java

```diff
@@ -170,11 +172,24 @@ public Response httpGetPersistenceServices(@Context HttpHeaders headers,
     public Response httpGetPersistenceServiceConfiguration(@Context HttpHeaders headers,
             @Parameter(description = "Id of the persistence service.") @PathParam("serviceId") String serviceId) {
         PersistenceServiceConfiguration configuration = persistenceServiceConfigurationRegistry.get(serviceId);
+        boolean editable = managedPersistenceServiceConfigurationProvider.get(serviceId) != null;
+
+        if (configuration == null) {
+            PersistenceService service = persistenceServiceRegistry.get(serviceId);
+            if (service != null) {
+                List<PersistenceStrategy> defaults = service.getDefaultStrategies();
+                List<PersistenceStrategy> strategies = defaults.stream()
```

Comment:
Why do you filter here? The `PersistenceManagerImpl` is applying a different configuration: 

https://github.com/openhab/openhab-core/blob/977fa0329a7d5473439845db3061e260328a6696/bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/internal/PersistenceManagerImpl.java#L451-L457

Reply:
My proposal was not adding the full (default configuration) to the REST response, only the default strategies and adding the list of cron strategies to the list of strategies (otherwise a cron default strategy would not work).

It is indeed properly better to respond with the full default configuration and further improve the UI (the UI currently does not allow selecting all items in the configuration, it will show it though, and it can be removed).